mkdocs.yml

```diff
@@ -129,7 +129,7 @@ plugins:
 nav:
   - Home:
     - About acLabs: index.md
-    - acLabs Quickstart: aclabs-quickstart.md
+    - Quickstart: quickstart.md
     - Codespaces Quickstart: codespaces-quickstart.md
```

Comment:
I'd suggest:
1) Updating the TechLib link to the new quickstart. Currently it's pointing to the old one.
2) Commenting the line for the old guide to avoid deploying it on pages, while not loosing the content.

Reply:
Good catch. Will open a PR on TechLib to resolve this today.

Reply:
Updated. Old guide commented out as well.